lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+
+// Returns true if by adding a term you can simplify part of the function into an integer
+// e.g. (2x+1)/(2x+3) -> True
+// e.g. (2x+1)/(2x^2 + 3) -> False
+function canFindDenominatorInNumerator(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '/' ) {
+    return false;
+  }
+  if (node.args.length !== 2) {
+    return false;
+  }
+
+  let numerator = node.args[0];
+  let denominator = node.args[1];
+  if (Node.Type.isParenthesis(numerator)) {
+    numerator = numerator.content;
+  }
+  if (Node.Type.isParenthesis(denominator)) {
+    denominator = denominator.content;
+  }
+
+  if (numerator.op !== '+') {
+    return false;
+  }
+  if (!('args' in denominator)) {
+    return false;
+  }
+  const numeratorFirstTerm = new Node.PolynomialTerm(numerator.args[0]);
```

Comment:
most of the time I think it'll be sorted so that the first term is the poly term, but I don't think this is guaranteed. you could have `(1+x) / (x+2)`. So we might need to have a step somewhere that sorts, or otherwise iterate through the arguments to find this poly term

Also, where do you make sure they have the same exponent? (i.e. for now that it's always 1)